webrtc.html

```diff
@@ -2775,8 +2775,39 @@
           <dd>
             <p>An array containing parameters for RTP encodings of media.</p>
           </dd>
+
+          <dt>sequence&lt;RTCRtpHeaderExtensionParameters&gt; headerExtensions</dt>
+          <dd>
+            <p>An array containing parameters for RTP header extensions.</p>
+          </dd>
+
+          <dt>RTCRtcpParameters</dt>
+
+          <dd>
+            <p>Parameters used for RTCP.</p>
+          </dd>
         </dl>
+
         <dl class="idl" title="dictionary RTCRtpEncodingParameters">
+          <dt>unsigned long ssrc</dt>
+          <dd>
+            <p>
+              The SSRC of the RTP source stream of this encoding
+              (non-RTX, non-FEC RTP stream).  Effectively readonly.
+              May not be changed by the application.
```

Comment:
I suggest just saying "readonly"  and then explaining in the text what happens if you try to change it.